apps/ex_wire/lib/ex_wire/kademlia/discovery.ex

```diff
@@ -15,6 +16,15 @@ defmodule ExWire.Kademlia.Discovery do
 
     this_round_nodes = RoutingTable.discovery_nodes(table)
 
+    _ =
```

Comment:
Since we're not using this variable, how would you feel about removing the assignment? i.e.

```ex
if Config.perform_sync?() do
  for node <- this_round_nodes do
    node
    |> Peer.from_node()
    |> PeerSupervisor.new_peer()
  end
end
```

Reply:
I believe I only added this due to a credo and/or dialyzer check.